src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -103,15 +107,28 @@ public Client BuildClient(OperationGroup operationGroup)
                     Paging pagingMethod = GetClientMethodPaging(processed.Method, nextPageMethod, pageable, objectType);
                     pagingMethods.Add(pagingMethod);
                 }
+                // For some reason, booleans in dictionaries are deserialized as string instead of bool.
+                bool longRunningOperation = Convert.ToBoolean(processed.Operation.Extensions.GetValue<string>("x-ms-long-running-operation") ?? "false");
+                if (longRunningOperation && pageable == null)
```

Comment:
Why do we need to check for `pageable == null`?

Reply:
https://github.com/Azure/autorest.csharp/issues/450
You have to write some kind of logic that combines LRO and paging. It has to actually change what is being called while doing paging operations. For now, that was outside the scope of this PR, and is something we can add to the generator when it is actually needed.

Reply:
Is LRO+Paging a real thing?

Reply:
In the test server, yes. In practice, no idea.